### ISSUE TITLE
Copy and Paste can be more smart in the VSCode alike the Pycharm?

### ISSUE THREAD

**Issue by gonghenghai** [microsoft/pylance-release#2240]
This is the behavior of Copy and Paste in the PyCharm:

![Animation](https://user-images.githubusercontent.com/21306413/149072043-a200b94c-007b-4c3c-bc90-7f311eff2ec3.gif)

This is the behavior of Copy and Paste in the VSCode:

![Animation2](https://user-images.githubusercontent.com/21306413/149072048-4ea6157d-0724-4377-88f9-3ed91e783186.gif)

We can find, it's smarter in the PyCharm. It can ignore the indent in the clipboard and put the function in a suitable place.

It's not provided by the formatter in PyCharm and the formatter in VSCode can not achieve it either.

**Comment by deterok** [comment]
Yeah! It would be great if you add smart copy paste to Python module

**Comment by heejaechang** [comment]
let us take a look. thank you for the suggestion!

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
